### PR TITLE
[#IOPLT-294] Adding Github Action configuration

### PR DESCRIPTION
**Pull request by drmarro**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - created
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v2
```
Comment (CrisTofani) [inline .github/workflows/release.yml:15]: i think we need to refer to the specific commit due to security requirements

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        with:
+          persist-credentials: false
+
+      - name: Log in to the Container registry
+        id: docker_login
+        uses: docker/login-action@v2
```
Comment (CrisTofani) [inline .github/workflows/release.yml:21]: ditto

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push Docker image
+        id: docker_build_push
+        uses: docker/build-push-action@v3
```
Comment (CrisTofani) [inline .github/workflows/release.yml:29]: ditto

**Review by CrisTofani — CHANGES_REQUESTED** [review]

File: .github/workflows/code-review.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      - name: Cache turbo build setup
+        uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 #v3.3.1
+        with:
+          path: node_modules/.cache/turbo
+          key: ${{ runner.os }}-turbo-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-turbo-
+
+      - name: 'Code review'
+        run: yarn run code-review
```
Comment (AleDore) [inline .github/workflows/code-review.yaml:46]: we must provide `code-review` script in `package.json` too

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+name: Release
```
Comment (AleDore) [inline .github/workflows/release.yml:1]: we can safely remove release pipeline at the moment

**Review by AleDore — APPROVED** [review]
lgtm

File: package.json [hunk 6]
```diff
@@ -12,7 +12,8 @@
     "start-ts": "ts-node -r dotenv/config src/index.ts",
     "start-test": "ts-node -r dotenv/config src/utils/test.ts",
     "start": "node -r dotenv/config dist/index.js",
-    "build": "tsc --outDir dist"
+    "build": "tsc --outDir dist",
+    "code-review": "yarn lint && yarn test"
```
Comment (AleDore) [inline package.json:16]: we must define "test" script :)

**Review by AleDore — CHANGES_REQUESTED** [review]

File: package.json (hunk 6 above)
Comment (drmarro) [reply package.json:16]: I've already added the "test" changes in this pull request -> https://github.com/pagopa/change-data-capturer-ms/pull/2

Comment (AleDore) [reply package.json:17]: We should provide test in this pull request ( so we can test pipeline directly in this PR )

Comment (drmarro) [reply package.json:17]: Ok fixed. The pipeline is green now.

**Pull request merged**